tests/unit/models/project-test.js

```diff
@@ -60,7 +60,7 @@ describe('models/project.js', function() {
 
       var expected = path.normalize('tests/dummy/config/environment');
 
-      expect(project.configPath()).to.equal(expected);
+      expect(project.configPath()).to.match(new RegExp(expected + '$'));
```

Comment:
We could. I thought the more specific assertion, that the expected path exists at the very end of the output, was a better check than the more generic one, that the expected path exists somewhere within the output.